### PR TITLE
[zh] Sync release notes for 1.20.4 and 1.19.8 #14704 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync release notes for 1.20.4 and 1.19.8 #14704 into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

**Comment by hanxiaop** [comment]
> ping @hanxiaop

Will take a look shortly, I'm going to publish the release soon.

File: content/zh/news/releases/1.19.x/announcing-1.19.8/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    这些限制适用于以下数据路径：
+    * Envoy 代理之间的 mTLS 通信。
+    * Envoy 代理下游和上游的常规 TLS（例如网关）
+    * 来自 Envoy 代理的 Google gRPC 端请求（例如 Stackdriver 扩展）。
+    * Istiod xDS 服务器。
+    * Istiod 用于注入和验证 Webhook 服务器。
+
+    这些限制不适用于以下数据路径：
+    * Istiod 到 Kubernetes API 服务器。
+    * JWK 从 Istiod 获取。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.19.x/announcing-1.19.8/index.md:32]:
```suggestion
    * 从 Istiod 获取 JWK。
```

File: content/zh/news/releases/1.20.x/announcing-1.20.4/index.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    这些限制适用于以下数据路径：
+    * Envoy 代理之间的 mTLS 通信。
+    * Envoy 代理下游和上游的常规 TLS（例如网关）
+    * 来自 Envoy 代理的 Google gRPC 端请求（例如 Stackdriver 扩展）。
+    * Istiod xDS 服务器。
+    * Istiod 用于注入和验证 Webhook 服务器。
+
+    这些限制不适用于以下数据路径：
+    * Istiod 到 Kubernetes API 服务器。
+    * JWK 从 Istiod 获取。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20.4/index.md:32]:
```suggestion
    * 从 Istiod 获取 JWK。
```

File: content/zh/news/releases/1.20.x/announcing-1.20.4/index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+- **修复** 修复了一个其中包含服务注册表中不存在的通配符主机的
+  `VirtualService` 被忽略的错误。
+  ([Issue #49364](https://github.com/istio/istio/issues/49364))
+
+- **修复** 修复了 `istioctl precheck` 报告与资源权限相关的 IST0141 消息不准确的问题。
+  ([Issue #49379](https://github.com/istio/istio/issues/49379))
+
+- **修复** 修复了仅使用 IPv6 集群的错误，
+  该错误阻止基于 ServiceEntry 的侦听器获得正确的 SNI 匹配。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20.4/index.md:53]:
```suggestion
  该错误阻止基于 `ServiceEntry` 的侦听器获得正确的 SNI 匹配。
```

**Review by hanxiaop — APPROVED** [review]

**Comment by hanxiaop** [comment]
@zirain Looks good

**Review by zirain — APPROVED** [review]

**Pull request merged**